src/app/series/directives/series-engagement.component.html

```diff
@@ -0,0 +1,24 @@
+<div *ngIf="!loading">
+  <prx-fancy-field label="Engagement Settings">
+    <div class="fancy-hint">Let your listeners know how they can interact with you further.</div>
+  </prx-fancy-field>
+
+  <div class="engagement" *ngIf="!podcast">
+    <h2>No podcast found - set it up on the Podcast Info tab first.</h2>
+  </div>
+
+  <div class="engagement" *ngIf="podcast">
+    <prx-fancy-field textinput [model]="podcast" name="donationUrl" label="Donation Form">
+      <div class="fancy-hint">
+        Let your listeners know where they can support your work by entering in an address to your donation form.
+      </div>
+    </prx-fancy-field>
+
+    <prx-fancy-field textinput [model]="podcast" name="paymentPointer" label="Micropayments Wallet">
+      <div class="fancy-hint">
+        The PRX Embeddable player support micropayments. We've created a guide to walk you through the process of collecting payments as
+        folks listen. <strong><a href="TODO">Learn more about micropayments.</a></strong>
```

Comment:
@brandonhundt where should this link go, for now?  Or should I leave it out?

Reply:
Leave it out for now. I need to talk to distro about this. I'll start that convo.